examples/readonly-objects/readonly_objects.bal

```diff
@@ -0,0 +1,46 @@
+import ballerina/io;
+
+// An object type descriptor.
```

Comment:
```suggestion
// An object-type descriptor.
```

Reply:
In this context, "type descriptors" are like a concept in Ballerina. We have type descriptors, and object type descriptors are a subcategory of the same.

Reply:
+1 to not use the hyphen.